### PR TITLE
Create file of githubteacher buenos aires file

### PR DESCRIPTION
**Pull request by francisco-cho**
File contains things to do in Buenos Aires, Argentina (not really)

- [ ] add places to avoid

- [ ] add places to seek

- [ ] add any content at all

### CODE REVIEW

**Comment by francisco-cho** [comment]
@kevinmalhotra @kkarpala

File: francisco-cho-buenos aires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Buenos Aires, Argentina
+
+### Places to avoid
```
Comment (nathyga) [inline francisco-cho-buenos aires]: Should I go to Caminito? is it safe?

Comment (francisvicta) [inline francisco-cho-buenos aires]: It's never safe...

**Comment by kevinmalhotra** [comment]
Where can I walk around shirtless?

**Pull request merged**